pandas/core/arrays/interval.py

```diff
@@ -657,8 +655,10 @@ def fillna(self, value=None, method=None, limit=None):
 
         self._check_closed_matches(value, name="value")
 
-        left = self.left.fillna(value=value.left)
-        right = self.right.fillna(value=value.right)
+        from pandas import Index
+
+        left = Index(self.left).fillna(value=value.left)
```

Comment:
umm why do you need to coerce to .fillna?

Reply:
ATM self.left is an ndarray which doesnt have fillna

Reply:
ahh